### PR TITLE
[stable/prometheus] Add RollingUpdate for Prometheus node-exporter ds

### PR DESCRIPTION
**Pull request by prat0318**
By default updateStrategy is `OnDelete` which needs manual deletion of all daemonsets for change to take affect

### CODE REVIEW

File: stable/prometheus/templates/node-exporter-daemonset.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ metadata:
     release: {{ .Release.Name }}
   name: {{ template "prometheus.nodeExporter.fullname" . }}
 spec:
+  updateStrategy:
```
Comment (dhilipkumars) [inline stable/prometheus/templates/node-exporter-daemonset.yaml]: This should be first added in `values.yaml` and then used here.

refer other sections such as `pod.spec.resources`, how they have been defined in the .values.yaml file and how they have been used here
```
 resources:
{{ toYaml .Values.nodeExporter.resources | indent 12 }}
```

Please bump update the chart version

**Review by dhilipkumars — CHANGES_REQUESTED** [review]

File: stable/prometheus/templates/node-exporter-daemonset.yaml (hunk 1 above)
Comment (prat0318) [reply stable/prometheus/templates/node-exporter-daemonset.yaml]: Made the changes. Also tested with `helm template`:
```
  updateStrategy:
    rollingUpdate:
      maxUnavailable: 1
    type: RollingUpdate
```

**Comment by prat0318** [comment]
signed CNCF

**Comment by prat0318** [comment]
@dhilipkumars bump on the PR

File: stable/prometheus/values.yaml [hunk 2]
```diff
@@ -238,6 +238,13 @@ nodeExporter:
     tag: v0.14.0
     pullPolicy: IfNotPresent
 
+  ## Set update Strategy to Rolling Update instead of default OnDelete
+  ##
+  updateStrategy:
+    type: RollingUpdate
```
Comment (mgoodness) [inline stable/prometheus/values.yaml]: Stick to the native default behavior: set `type: OnDelete` and comment the following lines.

**Review by mgoodness — CHANGES_REQUESTED** [review]

File: stable/prometheus/values.yaml (hunk 2 above)
Comment (prat0318) [reply stable/prometheus/values.yaml]: Fixed. I also removed the following lines and those comments will not make sense on OnDelete option.

**Comment by dhilipkumars** [comment]
@prat0318 apologies for the delay. Could you re-base the PR?

**Comment by prat0318** [comment]
@dhilipkumars done.

**Pull request merged**